src/airtable/rounds/funding_rounds.js

```diff
@@ -177,40 +180,42 @@ const calculateWinningProposalsForEarmark = (
       const multiplierOceanNew = (totalPool - fundedTillNow) / sumWeights
 
       proposals.map((x) => {
-        if (x.fields.funded !== x.fields.maxFund) {
+        if (!x.isFunded()) {
```

Comment:
Much simpler to read and verify, I can also see how it didn't include totalFund (previous history)